### DISCUSSION TITLE
Flood console

### DISCUSSION THREAD

**Discussion by oLoouiz** [thecoder-001/MineColab#111] [Q&A]
I used to use nrok, i discovered that playit.gg is better cause the ip doesn't change, so i set up it for the playit.gg with the plugin etc.
But now my server console is flooded, i can join the server normally, but why is it like it?
![imagem_2024-05-21_170930699](https://github.com/thecoder-001/MineColab/assets/82916508/6ce5941d-e55c-424a-a33d-1ce5f78703c2)

**Comment by N-aksif-N** [comment]
Hello, If you don't might, try this: https://github.com/N-aksif-N/MineColab_Improved
It's a improvement for minecolab
